test/app/ForgotPassword.spec.js

```diff
@@ -1,30 +0,0 @@
-import React from 'react';
```

Comment:
yeah, both those files have to live on the coral-auth server. I just forgot to delete them before.